### DISCUSSION TITLE
APT repository, PPA, DEB, Flatpak or AppImage

### DISCUSSION THREAD

**Discussion by KaKi87** [sxyazi/yazi#1751] [4. Ideas]
Hello,

Having any of those for easier installation and updates on Debian-based distros would be nice.

Thanks

**Comment by alextopic** [comment]
Ya, I guess they don't like Debian/Ubuntu. I guess you could build from sources ,if you wanted.

**Reply by KaKi87** [reply]
> if you wanted

I don't :sweat_smile:

**Reply by alextopic** [reply]
Well you can't expect everyone to be really familiar with every Linux distro. When I get some more time, I might investigate further..

**Reply by KaKi87** [reply]
This is not about knowing every distro, but knowing the *top* **family** of distros, which they can't not. Also, Flatpak and AppImage aren't specific to Debian-based distros.

**Comment by sxyazi** [comment]
I would love to support them, but I'm not sure how to do it since Debian/Ubuntu isn't my work system, and I hardly use them. 

Any help would be welcome and greatly appreciated! 🙏🏻

**Reply by AnirudhG07** [reply]
You can try and use `cargo deb` command, which u can download with `cargo install cargo-deb`.
Run the commands, `cargo deb -p yazi-cli` or `cargo deb -p yazi-fm` (before this simply run `cargo deb`).
These will create `.deb` files, which you need to give to some server. One way is [LaunchPad](https://launchpad.net). If you can other ways, great!

**Comment by sxyazi** [comment]
Yazi now supports Snapcraft and Flatpak:

- https://yazi-rs.github.io/docs/installation/#snapcraft
- https://yazi-rs.github.io/docs/installation/#flatpak

**Reply by KaKi87** [reply]
Thanks, I also see there's `brew` now, apparently it's been a while though, I don't know how I missed it in the documentation.

**Comment by canh25xp** [comment]
There's a guy create an unofficial debian repository that include yazi

https://github.com/dariogriffo/yazi-debian

it doesn't contain the ya Package Manager though

**Reply by KaKi87** [reply]
I get `NO_PUBKEY 7808B4DD62C41256` though, despite running the curl|gpg command 🤔

**Reply by canh25xp** [reply]
Public key changed; you can follow the new instructions at this site: https://debian.griffo.io/

**Reply by KaKi87** [reply]
That worked, thank you. It would have been nice of them to update the README instructions though.

Also, it seems like arm64 isn't supported, so I'll have to keep using Flatpak on that one.
